### ISSUE TITLE
Dialogues are too big and not resizable

### ISSUE THREAD

**Issue by awsdert** [RPCS3/rpcs3#7420]
RPCS3 Version: 0.0.8-9525-17df6c88
The Dialogues don't fit in my resolution (1920x1080), this is no doubt because RPCS3 doesn't understand that I chose large fonts to make them easier to read on my projector

![Screenshot from 2020-02-07 10-15-21](https://user-images.githubusercontent.com/7422982/74021156-facccc80-4992-11ea-9e66-123fc8aa5fa9.png)

Please make ALL dialogues use a scrollbar with a smaller default size which can then be resized to a larger size by mouse if one so desires, the scrollbar can just auto hide if everything can be fit with the set window size

**Comment by Megamouse** [comment]
buy glasses

**Comment by awsdert** [comment]
Its not glasses I need, a projector doesn't give sharp text at  small sizes so large size is a must, I don't want to shove my face half a meter from a screen that glares light at me, it just hurts my eyes, a projector doesn't since the light is always coming from behind. Also it shouldn't take that much effort to just add a few lines of code to the start of every dialogue creator to shove a scroll frame in

**Comment by elad335** [comment]
thats really not rpcs3 fault, use builtin Windows' maginifier tool.

**Comment by MSuih** [comment]
it's an issue with rpcs3 but just low priority due to very few people being affected by this

**Comment by awsdert** [comment]
> thats really not rpcs3 fault, ~use builtin Windows' maginifier tool~ I mean similar program on the internet.

I know text size isn't rpcs3's fault, but unresizable dialogs are, please, all I'm asking is that you shove in the flag for resizable and have everything use a minimum size for when the window is sized smaller than the default (so after calculating the needed size override if less than minimum), that should be enough for me to fit on my screen, albiet with difficulty looking at the info for options (which I can just lookup online)

**Comment by Megamouse** [comment]
Pad settings dialog resize should work after #8553is merged

**Comment by awsdert** [comment]
ty, now I just need to make sure mines upto date again, last time I tried for some reason it would not compile completely, I think it should be rectifiable on your end if you rename your current makefile to something like main.mak and then just subspawn make with correct overrides (like -j 1 -f main.mak) via a new makefile
